### PR TITLE
Add crd & rbac rule for ClusterClaim controller

### PR DESCRIPTION
**Pull request by elgnay**
Signed-off-by: Yang Le <yangle@redhat.com>

The task link: https://github.com/open-cluster-management/backlog/issues/7158

### CODE REVIEW

**Comment by qiujian16** [comment]
do we remove the crd when klusterlet is deleted?

**Comment by elgnay** [comment]
> do we remove the crd when klusterlet is deleted?

The logic is added to delete ClusterClaim crd once klusterlet is deleted.

File: pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_controller.go [hunk 1]
```diff
@@ -38,11 +38,13 @@ const (
 	klusterletFinalizer = "operator.open-cluster-management.io/klusterlet-cleanup"
 	imagePullSecret     = "open-cluster-management-image-pull-credentials"
 	klusterletApplied   = "Applied"
+	clusterClaimCrdFile = "manifests/klusterlet/0000_02_clusters.open-cluster-management.io_clusterclaims.crd.yaml"
```
Comment (qiujian16) [inline pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_controller.go]: should we just put this in staticResourceFiles?

Comment (elgnay) [reply pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_controller.go]: Yes, we can. I've updated the code as suggested.

**Pull request merged**